packages/react-instantsearch-nextjs/package.json

```diff
@@ -49,7 +49,7 @@
   },
   "devDependencies": {
     "instantsearch.js": "4.56.11",
-    "next": "13.4.19",
+    "next": "13.5.1",
     "react-instantsearch-core": "7.0.3"
   },
   "peerDependencies": {
```

Comment:
do we not need to bump peer too?

Reply:
Not necessarily but it's best we keep one version of Next in `node_modules`

Reply:
I could have kept the examples on `13.4` but `13.5` builds faster so that's nicer for CI :D